### PR TITLE
Add the update dataset documentation

### PR DESCRIPTION
**Pull request by Dexterp37**
This PR adds the documentation for the newly added update ping dataset.

### CODE REVIEW

File: concepts/choosing_a_dataset.md [hunk 1]
```diff
@@ -68,6 +68,17 @@ This data is available in the **telemetry_new_profile_parquet** dataset.
 
 {% include "/datasets/batch_view/new_profile/intro.md" %}
 
+# Update Derived Dataset
+
+The [update ping](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/update-ping.html)
```
Comment (harterrt) [inline concepts/choosing_a_dataset.md]: This summary of the update ping is super useful. Do you mind moving lines 73-76 into intro.md? That way this summary will be included in the reference documentation as well.

File: concepts/choosing_a_dataset.md [hunk 2]
```diff
@@ -68,6 +68,17 @@ This data is available in the **telemetry_new_profile_parquet** dataset.
...
+is sent from Firefox Desktop when a browser update is ready to be applied and after it was correctly applied.
+It contains the build information and the update blob information, in addition to some information about the
+user environment.
+
+This data is available in the **telemetry_update_parquet** dataset.
```
Comment (harterrt) [inline concepts/choosing_a_dataset.md]: No need for this line. Accessing the data is discussed in intro.md, which is included below.

**Comment by Dexterp37** [comment]
Thanks Ryan!

**Comment by Natim** [comment]
Apparently this is not yet readable on http://docs-origin.telemetry.mozilla.org/ any actions required?

**Comment by harterrt** [comment]
I just pushed the docs manually. Looks like it's there now.

**Pull request merged**